### PR TITLE
FC-1340 FC-1342 feat(adyen): added supported list of payment methods

### PR DESCRIPTION
**Pull request by Kagaminara**
Also added steps in order to enable  Google Pay.

The supported list features Alipay and ApplePay as well, that's under the condition Adyen support successfully helps us with the issue we have before the end of the cycle.

### CODE REVIEW

File: docs/advanced/payments/adyen.mdx [hunk 1]
```diff
@@ -470,3 +516,17 @@ the Cart amount and the Store currency. Different contexts could display
 different payment methods in the checkout.
 
 :::
+
+# Payment methods support
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:520]:
```suggestion
## Payment methods support
```

File: docs/advanced/payments/adyen.mdx [hunk 2]
```diff
@@ -470,3 +516,17 @@ the Cart amount and the Store currency. Different contexts could display
...
+The following payment methods have been tested:
+
+- Credit card
+- Paypal
+- Klarna (Pay now, Pay over time, Pay later)
+- Google Pay
+- ApplePay
+- Sofort
+- Alipay: There is a known issue with Alipay when integrating Adyen through the
+  Magento 2 module
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:531]: is this comment about the adyen support request currently going on ?

Comment (Kagaminara) [reply docs/advanced/payments/adyen.mdx:531]: Yes. I wasn't planning on merging this until we have a proper answer (or the cycle ends).

File: docs/advanced/payments/adyen.mdx (hunk 1 above)
Comment (Kagaminara) [reply docs/advanced/payments/adyen.mdx:520]: I found it a bit weird to have this on the same level as the other "As a XXX", but I can go with this.

File: docs/advanced/payments/adyen.mdx (hunk 2 above)
Comment (Kagaminara) [reply docs/advanced/payments/adyen.mdx]: Confirmed it works after Adyen manual configuration. Updated the docs accordingly.

File: docs/advanced/payments/adyen.mdx [hunk 3]
```diff
@@ -470,3 +516,16 @@ the Cart amount and the Store currency. Different contexts could display
...
+
+## Payment methods support
+
+The following payment methods have been tested:
+
+- Credit card
+- Paypal
+- Klarna (Pay now, Pay over time, Pay later)
+- Google Pay
+- ApplePay
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:528]:
```suggestion
- Apple Pay
```

(according to https://www.apple.com/fr/apple-pay/)

**Review by dpobel — APPROVED** [review]

**Pull request merged**